examples/polarify/pixi.toml

```diff
@@ -52,7 +52,7 @@ py39 = ["py39", "test"]
 py310 = ["py310", "test"]
 py311 = ["py311", "test"]
 py312 = ["py312", "test"]
-lint = ["lint"]
+lint = { features=["lint"], no-default-feature=true}
```

Comment:
Hum, this does not seem to work. I get the following error. not sure what is happening

```bash
  × failed to solve the conda requirements of 'lint' 'linux-64'
  ╰─▶ Cannot solve the request because of: No candidates were found for pre-commit *.
```

Reply:
Note that specifying the same solve-group for the `lint` and `default` environment fixes the issue.

There seem to be a bug there. Maybe the same [I hit where python was not detected when creating environments from optional extras ?](https://github.com/prefix-dev/pixi/pull/1077#discussion_r1544058775)

Reply:
Haha, just debugged the issue as I also couldn't think of a reason. Pixi was doing exactly what we're asking, it didn't have any channels as those are also part of the "default feature". I'm not sure how we want to deal with this. Adding this fixes it:
```toml
[feature.lint]
channels = ["conda-forge"]
```

I see some options, either the user should add the channels and we add a better error that tells the user that we don't add any channels to the environment in question. Or we add the channels from the default feature anyway. I guess the later is more comfortable but it's also kinda wrong. Especially with multi channel environments.